convert2rhel/actions/pre_ponr_changes/handle_packages.py

```diff
@@ -162,12 +169,13 @@ def run(self):
             self.add_message(
                 level="INFO",
                 id="SPECIAL_PACKAGES_REMOVED",
-                title="Special packages to be removed",
-                description=(
+                title="Packages to be removed",
+                description=message,
+                diagnosis=(
                     "We have identified installed packages that match a pre-defined list of packages that are"
-                    " to be removed during the conversion"
+                    " known to cause a conversion failure."
                 ),
-                diagnosis=message,
+                remediations=("Check that the system runs correctly without the packages after the conversion."),
```

Comment:
@bookwar, do you see a better action the user can take?

Reply:
```suggestion
                remediations="Check that the system runs correctly without the packages after the conversion.",
```